packages/valory/skills/safe_deployment_abci/tests/test_contract_api.py

```diff
@@ -22,36 +22,37 @@
 # pylint: skip-file
 
 import asyncio
```

Comment:
Why the changes?

Reply:
Although the `gnosis_safe_contract` fixture was defined on `ledger` connection we didn't use it anywhere else other than here so when merging the ledger connection I moved that fixture here